pyproject.toml

```diff
@@ -64,9 +64,8 @@ show_missing = true
 fail_under = 60
 
 [tool.djlint]
-# TODO: Enforce viewport?
```

Comment:
Mainly just want to remove this line because having a viewport tag is already enforced by djlint